### ISSUE TITLE
Missing characters when using Google Fonts.

### ISSUE THREAD

**Issue by hand-dot** [pdfme/pdfme#124]
### Describe the bug

When I use one of the google fonts as custom font, the designer can render correctly, but the generator can't render some characters.

<img width="1145" alt="CleanShot 2022-12-31 at 16 42 00@2x" src="https://user-images.githubusercontent.com/24843808/210129384-c2961e0a-addd-467d-b203-acffc4036544.png">
<img width="1210" alt="CleanShot 2022-12-31 at 16 42 19@2x" src="https://user-images.githubusercontent.com/24843808/210129404-e59812e5-4bae-4201-bb77-75e5b8858b80.png">
(↑missing 'e' and 'x' of 'text' )

### To Reproduce

Use the below font as a custom font.

- Font: https://fonts.google.com/specimen/Zen+Kurenaido?subset=japanese&noto.script=Jpan
- Font URL: http://fonts.gstatic.com/s/zenkurenaido/v12/3XFsEr0515BK2u6UUptu_gWJZfz22PRLd0U.ttf
- About custom font: https://pdfme.com/docs/guides/custom-font

### Expected behavior

Must be able to generate PDFs without missing characters.

### Your Environment

```markdown
- pdfme package(@pdfme/generator or @pdfme/ui): @pdfme/generator
- pdfme version: 1.0.18
- Operating system: MacOS
- Node.js version or Browser name & version: Google Chrome: 108.0.5359.124, Node: 16.18.1
```

### Your Error Log

```shell
No Error Log
```

**Comment by hand-dot** [comment]
ref: https://github.com/foliojs/pdfkit/issues/1357

**Comment by hand-dot** [comment]
I found this problem was caused by a font file problem (huge font such as Japanese and Chinese).

I commented on relevant issues at pdfkit (https://github.com/foliojs/pdfkit/issues/1357#issuecomment-1386375205), pls check that.

So, If we use Japanese and Chinese,  we have to use `ttf` font file.
(Maybe it's a lack of font feature? I'm not sure...)

Anyway, when we want to fix this problem, we have to commit to the [foliojs/fontkit](https://github.com/foliojs/fontkit) and [Hopding/fontkit](https://github.com/Hopding/fontkit).

So, I'll close this issue.
